File: app/src/main/java/com/example/nickolas/vk/models/remote/IDialogsDataSource.java
package com.example.nickolas.vk.models.remote;

import okhttp3.ResponseBody;
import rx.Observable;

/**
 * Created by Nickolas on 01.10.2017.
 */

public interface IDialogsDataSource {

    Observable<ResponseBody> getDialogs(int offset, String tokken);
}

File: app/src/main/java/com/example/nickolas/vk/models/remote/IDialogsDataSource.kt
package com.example.nickolas.vk.models.remote

import com.example.nickolas.vk.models.enteties.Dialog

import okhttp3.ResponseBody
import rx.Observable

/**
 * Created by Nickolas on 01.10.2017.
 */

interface IDialogsDataSource {

    fun getDialogs(offset: Int, token: String): Observable<ResponseBody>
    fun getUsersData(dialogs: List<Dialog>): Observable<ResponseBody>

}
